src/Evolution/Executables/GeneralizedHarmonic/EvolveGhSingleBlackHole.hpp

```diff
@@ -135,10 +137,12 @@ struct EvolutionMetavars : public GeneralizedHarmonicTemplateBase<VolumeDim> {
     using interpolating_component = typename metavariables::gh_dg_element_array;
   };
 
-  using control_systems = tmpl::list<control_system::Systems::Shape<
-      ::domain::ObjectLabel::None, 2,
-      control_system::measurements::SingleHorizon<
-          ::domain::ObjectLabel::None>>>;
+  using control_systems =
+      tmpl::list<control_system::Systems::Shape<
+                     ::domain::ObjectLabel::None, 2,
+                     control_system::measurements::SingleHorizon<
+                         ::domain::ObjectLabel::None>>,
+                 control_system::Systems::Size<::domain::ObjectLabel::None, 2>>;
```

Comment:
So I understand, why is shape templated on measurement while size is not?

Reply:
Size control always uses the same measurement, `control_system::measurements::CharSpeed`, for both single BH and BBH. Shape control uses different measurements: For single BH it uses `control_system::measurements::SingleHorizon` whereas for BBH it uses `control_system::measurements::BothHorizons` because shape is with all other slow control systems. So shape has to be templated on the measurement because it uses different ones for single BH vs BBH, while size is always the same one.